appveyor.yml

```diff
@@ -4,11 +4,21 @@ os: Visual Studio 2015
 
 environment:
   matrix:
+    - compiler: msvc-15-seh
+      generator: "Visual Studio 15 2017"
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+
+    - compiler: msvc-15-seh
+      generator: "Visual Studio 15 2017 Win64"
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+
     - compiler: msvc-14-seh
       generator: "Visual Studio 14 2015"
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
```

Comment:
I am not very familar with AppVeyor setup. Could you please explain "APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017" - I see that you added it to VC14 and 15 as well....

Reply:
@KindDragon

Reply:
This override build worker images from line `os: Visual Studio 2015`.  Only `Visual Studio 2015` image contains 2008-2017, but image `Visual Studio 2017` contains VS2017 https://www.appveyor.com/docs/build-environment/#visual-studio-2017

`Visual Studio 2017` image not required for VS2015, this image just smaller